images/aws-ocp-snap/include/create_snapshot.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+NAMESPACE=$1
+VOLUME=$2
+
+TMPFILE=$(mktemp /tmp/tmp.XXXXXXXXXXXXX)
+CHKFILE=$(mktemp /tmp/chk.XXXXXXXXXXXXX)
+
+if [ -z "$NAMESPACE" -o "$NAMESPACE" = " " ]; then
+    echo "A Namepace must be provided or select ALL"
+    exit 2
+fi
+case "$NAMESPACE" in
+        "ALL")
+            oc get pv --no-headers 2> $CHKFILE
+            RESULT=$?
+            NOVOL=$(grep "No resources" $CHKFILE | wc -l)
+            if [ ! $RESULT -eq 0 ] || [ $NOVOL -gt 0 ]; then
+                echo "Error while getting EBS volumes information"
+                exit 2
```

Comment:
This should not exit with an error code as the job controller will try to reschedule more pods. It should be treated as a success as there were no PV's to backup

Reply:
done